docs/integrations/datadog/event-destination.md

```diff
@@ -59,7 +59,7 @@ To send the events to Datadog we'll need to assign an Event Destination to the E
 
 1. Choose Datadog as the target and fill in the correct information being mindful of **which Datadog site your data resides in**.  Target, Region and API Key values are required, all other values are optional.
 
-    - **Service Name** is a locally significant value and will be appended to the event destination name.
+    - **Service Name** will be added as a key-value pair to the event payload as **service:value**.<br></br>
```

Comment:
Are the newlines here intentional?

Reply:
Yes, the service name is actually part of the payload going into datadog.  It's a searchable key.